domains/POAS/events/lpa-access-granted/schema.json

```
@@ -0,0 +1,28 @@
+{
+  "$id": "https://opg.service.justice.gov.uk/opg.poas.use/lpa-access-granted.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "opg.poas.use/lpa-access-granted",
+  "type": "object",
+  "properties": {
+    "uid": {
+      "type": "string",
+      "description": "The UID of the LPA",
+      "pattern": "^M(-[A-Z0-9]{4}){3}$"
+    },
+    "lpaType": {
+      "type": "string",
+      "description": "The type of LPA",
+      "enum": ["personal-welfare", "property-and-affairs"]
+    },
+    "actors": {
+      "type": "object",
+      "properties": {
+        "actorUid": {
+          "type": "string"
+        }
+      },
+      "required": ["actorUid"]
+    }
```

JSON schema doesn't have an easy way of defining maps. The best you can do is the slightly vague:
```suggestion
    "actors": {
      "type": "object",
      "additionalProperties": {"type": "string"}
    }
```

Perhaps a safer option would be to make them an array of objects: `{"actorUid": "9ac5cb7c-fc75-40c7-8e53-059f36dbbe3d", "authIdentifier": "urn:fdc:gov.uk:2022:XXXX-XXXXXX"}`. Which would make this:
```suggestion
    "actors": {
      "type": "array",
      "items": {
        "type": "object",
        "required": ["actorUid", "authIdentifier"],
        "properties": {
          "actorUid": {
            "type": "string",
            "format": "uuid"
          }
          "authIdentifier": {
            "type": "string"
          }
        }
      },
      "required": ["actorUid"]
    }
```